src/index.ts

```diff
@@ -171,6 +171,11 @@ class Monika extends Command {
       description: 'Write monika config file to this file',
       multiple: false,
     }),
+
```

Comment:
i think adding a --force flag is not solving the user experience issue. 

A better experiece may be:
"file monika.json already exist, overwrite (n/Y)"  ==> defaults to no if any other key pressed
This continues the command and at the same time alerts the user.
If the user click No/cancle, print out a message that the "-o" output option exist.

Failing the command  "file already exist!" and then forching him (or me) to retype the whole long commadn again AND add a new flag is not a good experience. AND we have to add another flag.

Reply:
Agreed.
This `force` command is too ambiguous. What is the thing that is forced?

Reply:
You don't need to retype the whole long command, just press the up arrow keyboard 🤣  but I don't mind with the Y/n. But `force` is still needed when user runs the command in automated environment like CI.

Reply:
If force command is ambiguous, maybe we can change it to --force-overwrite